src/Infiltrator.jl

```diff
@@ -108,8 +114,9 @@ function show_help(io)
     The following commands are special cased:
       - `@trace`: Print the current stack trace.
       - `@locals`: Print local variables.
+      - `@stop`: Stop infiltrating at this `@infiltrate` spot.
 
-    Exit this REPL mode with `Ctrl-D`.
+    Exit this REPL mode with `Ctrl-D`, and clear the effect of `@stop` with `Infiltrator.clear_stop()`.
```

Comment:
The regression tests aren't updated to this change yet, so I'm afraid you'll have to run them again.

Reply:
Ah right! By the way, when I run tests locally and they fail, I see a bunch of white squares instead of the diff between what is expected and what is obtained... did you hear about this problem?

Reply:
Yeah, that's happening because the normal TerminalRegressionTests is kinda broken, and [travis therefore explicitly uses a different branch](https://github.com/JuliaDebug/Infiltrator.jl/blob/master/.travis.yml#L23).